tutorials/Optimizer_Tutorial.ipynb

```diff
@@ -1172,24 +635,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### we also have returned to you the phoenics object. One interesting object we can extract from this is the acquisition function. You can obtain this indirectly, using resut.object.bayesian_network.kernel_contribution. This function takes a numpy array  ( a point in your parameter space) and returns 2 numbers, x and y; the acquisition function then has the value x*y. Note: this is often zero.\n",
+    "We also have returned to you the phoenics object. One interesting object we can extract from this is the acquisition function. You can obtain this indirectly, using resut.object.bayesian_network.kernel_contribution. This function takes a numpy array  ( a point in your parameter space) and returns 2 numbers, x and y; the acquisition function then has the value x*y. Note: this is often zero.\n",
```

Comment:
from that comment its unclear what the aquisition function is and what to do with that